docs/getting_started.rst

```diff
@@ -26,21 +26,17 @@ To draw a sky image from HiPS image tiles with the `hips` package, follow the fo
          coordsys='galactic', projection='AIT',
     )
 
-
 2. Specify the HiPS survey you want by creating a `~hips.HipsSurveyProperties` object.
 
    A good address that lists available HiPS data is http://aladin.u-strasbg.fr/hips/list ::
 
     from hips import HipsSurveyProperties
-    url = 'http://alasky.unistra.fr/DSS/DSS2Merged/properties'
-    hips_survey = HipsSurveyProperties.fetch(url)
-
+    hips_survey = HipsSurveyProperties.make('CDS/P/DSS2/red')
```

Comment:
If you remove the import and just write this here
```
hips_survey = 'CDS/P/DSS2/red'
```
it would give the same result, no?

Please make it work that way and change the high-level docs to use that simple form.